paraatm/bdn/_bdn.py

```diff
@@ -0,0 +1,157 @@
+        model.add(LSTM(self.num_unit,kernel_regularizer=l2(self.weight_decay),
+                       recurrent_regularizer=l2(self.weight_decay),
+                       bias_regularizer=l2(self.weight_decay),dropout=self.idrop,
+                       recurrent_dropout=self.rdrop,input_shape=(None, in_shape),
+                      kernel_initializer='random_uniform',return_sequences=True))
+
+        model.add(Activation('relu'))
+
+        model.add(LSTM(self.num_unit,dropout=self.idrop,
+                       recurrent_dropout=self.rdrop,return_sequences=False,
+                       kernel_regularizer=l2(self.weight_decay),
+                       recurrent_regularizer=l2(self.weight_decay),
+                       bias_regularizer=l2(self.weight_decay)))
+        model.add(Activation('relu'))
+        if self.odrop:
+            model.add(Dropout(self.odrop))
+        model.add(Dense(1,activation='linear',
+                        kernel_regularizer=l2(self.weight_decay),
+                        bias_regularizer=l2(self.weight_decay)))
+        optimizer_=Adam(self.lr)
+        model.compile(loss='mse',metrics=['mse'],optimizer=optimizer_)
+        return model
+    def get_DNN_model(self):
+
+        if len(self.x_train.shape)!=2:
+            raise Exception('expect a 2 dimension array!')
+        in_shape = self.x_train.shape[-1]
+        model=Sequential()
+        model.add(Dense(self.num_unit,kernel_regularizer=l2(self.weight_decay),
+                       bias_regularizer=l2(self.weight_decay),
+                        kernel_initializer='random_uniform',
+                        input_dim=in_shape))
+        
+        model.add(Activation('relu'))
+        model.add(Dropout(self.idrop))
+        model.add(Dense(self.num_unit,kernel_regularizer=l2(self.weight_decay),
+                       bias_regularizer=l2(self.weight_decay),
+                        kernel_initializer='random_uniform',
+                       ))
+        model.add(Dropout(self.idrop))
+        model.add(Activation('relu'))  
+        if self.odrop:
+            model.add(Dropout(self.odrop))
+        model.add(Dense(1,activation='linear',
+                        kernel_regularizer=l2(self.weight_decay),
+                        bias_regularizer=l2(self.weight_decay)))
+        optimizer_=Adam(self.lr)
+        model.compile(loss='mse',metrics=['mse'],optimizer=optimizer_)
+        return model        
+        
+
+    def BDNpred(self,flag_1 = True,flag_2 = 'RNN',custom_model = None):
+        """
+        BDNpred is used to generate results for the deep learning model
+        
+        Attributes
+        ----------
+        flag_1==True: prediction with uncertainty
+        flag_1==False: deterministic result
+        
+        flag_2 == 'RNN': generate results for the embedded RNN model
+        flag_2 == 'DNN': generate results for the embedded DNN model
+        flag_2 == 'custom': generate results for the customized model
+        
+        """
```

Comment:
Please add a `Parameters` section to the docstring to document the parameters.  This should list each parameter, its data type, and a short description.  See [iff.py](../../blob/master/paraatm/io/iff.py) for examples.